style.css

```diff
@@ -74,6 +74,23 @@ body{
 .officers-body {
     padding: 30px 0px 30px 0px;
 }
+
+.calendar-container {
+    position: relative;
+    padding-bottom: 75%;
```

Comment:
This causes problems on mobile AND desktop. 
![Screenshot](https://user-images.githubusercontent.com/15039983/62010530-04e75600-b13a-11e9-95c4-79780d7fcf1c.png)

Reply:
What would be the best way to fix this ?

Reply:
@TheOtherUnknown

Reply:
Letting Bootstrap take care of it would be ideal. We may need to modify the iframe itself.